<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FireMap</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="mainContent" stretch="0">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item alignment="Qt::AlignLeft">
       <widget class="QFrame" name="left_side_content">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label">
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="listOfLayers">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QListWidget::item {
 	border: 0px;
    padding-left: 30px;
 }
QListWidget::item:selected{
  color: black;
  border: 0px;
  background-color:rgb(244, 244, 244);
}</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="lineWidth">
            <number>0</number>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="defaultDropAction">
            <enum>Qt::IgnoreAction</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionNew_project"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAdd_layer">
    <property name="title">
     <string>Слой</string>
    </property>
    <addaction name="actionAdd_vector_layer"/>
    <addaction name="actionAdd_raster_layer"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>Вид</string>
    </property>
    <addaction name="actionShow_list_of_layers"/>
   </widget>
   <widget class="QMenu" name="menuModel">
    <property name="title">
     <string>Модель</string>
    </property>
    <addaction name="actionStart_model"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuAdd_layer"/>
   <addaction name="menuView"/>
   <addaction name="menuModel"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew_project">
   <property name="text">
    <string>Новый</string>
   </property>
  </action>
  <action name="actionOpen_project">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="actionSave_project">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="actionAdd_vector_layer">
   <property name="text">
    <string>Добавить векторный слой</string>
   </property>
  </action>
  <action name="actionAdd_raster_layer">
   <property name="text">
    <string>Добавить растровый слой</string>
   </property>
  </action>
  <action name="actionShow_list_of_layers">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Список слоев</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Выйти</string>
   </property>
  </action>
  <action name="actionStart_model">
   <property name="text">
    <string>Начать моделирование</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
